torch/csrc/jit/frontend/schema_matching.cpp

```diff
@@ -303,6 +303,36 @@ bool isBlockListedSchema(const FunctionSchema& schema) {
   return false;
 }
 
+int calculateNecessaryArgs(
```

Comment:
```suggestion
int CalculateNecessaryArgs(
```
I think we should try to follow Google C++ style guide as much as possible:
https://google.github.io/styleguide/cppguide.html#Function_Names